### PR TITLE
build/tools/amebad : Update J-link README.md

### PR DESCRIPTION
**Pull request by jeongchanKim**
1) Modify J-link and AmebaD connections
2) Add the path for clarification

### CODE REVIEW

**Review by an4967 — APPROVED** [review]
LGTM

**Review by sangwon03 — DISMISSED** [review]
LGTM

**Review by jeongarmy — APPROVED** [review]
LGTM

**Review by jsdosa — DISMISSED** [review]
looks nice to me.

File: build/tools/amebad/gnu_utility/README.md [hunk 1]
```diff
@@ -2,28 +2,35 @@
...
-=============================
+1. Prepare J-Link and Install JLinkGDBServer.  
+Download JLinkGDBServer from this [link](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).  
+After download, install through below command.  
+```
+$ sudo dpkg -i JLink_Linux_V686g_x86_64.deb
+```
+
+2. Upgrade the firmware to V10.  
+In 'GDB Server' window, there is firmware version like below.
```
Comment (sunghan-chang) [inline build/tools/amebad/gnu_utility/README.md:13]: How about below?
Upgrade the firmware to V10 and verify through the message of 'GDB Server' window like below.

File: build/tools/amebad/gnu_utility/README.md [hunk 2]
```diff
@@ -2,28 +2,35 @@
...
 ### Preparations
 
-1. Prepare J-Link with firmware upgraded to V10.
-2. Install JLinkGDBServer.
-3. Set 'environment variable' for JLinkGDBServer if not done so.
-
-=============================
+1. Prepare J-Link and Install JLinkGDBServer.  
+Download JLinkGDBServer from this [link](https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack).  
+After download, install through below command.
```
Comment (sunghan-chang) [inline build/tools/amebad/gnu_utility/README.md:7]: Download JLinkGDBServer from the link and install as below.

File: build/tools/amebad/gnu_utility/README.md [hunk 3]
```diff
@@ -2,28 +2,35 @@
...
-
-=============================
+| J-Link | AmebaD |
+|--------|--------|
+|   VCC  |  3.3v  |
+|   GND  |  GND   |
+|  SWDIO |  DATA  |
+|  SWCLK |  CLK   |
+  
+Refer to this [link](https://www.segger.com/products/debug-probes/j-link/technology/interface-description/).
```
Comment (sunghan-chang) [inline build/tools/amebad/gnu_utility/README.md:27]: Refer to J-Link interface information .

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**